packages/kbn-search-api-panels/components/code_box.tsx

```diff
@@ -62,6 +62,26 @@ export const CodeBox: React.FC<CodeBoxProps> = ({
 }) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false);
 
+  const getAriaLabel = () =>
```

Comment:
nit: there is no need to wrap it in a function if no arguments are passed.

Reply:
I'm going to be a little contrarian and ask if we can consider a different way of constructing an accessible label for these buttons. There's at least five of these dropdowns on the page and when I started a local serverless instance, and changing the programming language in one dropdown updates the other four.

I'm proposing we use the heading text, the programming language name, and some free text to create labels that are unique and contain the visible button text. Having the visible text in an accessible label is also a WCAG a requirement. See [SC 2.5.3: Label In Name (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html) for a deep dive on that point.

Let's follow this example screenshot to create accessible labels that are meaningful to the current code snippet and follow the heading text:

<img width="1187" alt="Screenshot 2024-07-23 at 1 58 30 PM" src="https://github.com/user-attachments/assets/ee2c5038-8e5c-4a14-a015-266fe2e1562f">